example.py

```diff
@@ -18,21 +18,21 @@ def create_accounts(conn):
             "CREATE TABLE IF NOT EXISTS accounts (id INT PRIMARY KEY, balance INT)"
         )
         cur.execute("UPSERT INTO accounts (id, balance) VALUES (1, 1000), (2, 250)")
-        logging.debug("create_accounts(): status message: %s", cur.statusmessage)
+        logging.debug(f"create_accounts(): status message: {cur.statusmessage}")
```

Comment:
It doesn't make too much of a change here, however, logging using `%s` and taking parameters separately is a performance feature. If logging is disabled (because e.g. it's set at INFO level) the conversion of the arguments to strings and the string concatenation is not performed. This way, it is always performed, and the string produced is discarded.

f-strings may be fashionable but here they are an inferior choice.

Reply:
Pylint is full of... well, let's be gentleman.